kie-api/src/main/java/org/kie/api/runtime/process/ProcessRuntime.java

```diff
@@ -48,6 +48,15 @@
     ProcessInstance startProcess(String processId,
                                  Map<String, Object> parameters);
 
+    /**
+     * Starts a new process instances starting from the nodes given
+     * @param processId the id of the process that should be started
+     * @param params the process variables that should be set when starting the process instance
+     * @param nodeInstancesIds the node instances id that should be triggered
```

Comment:
Please change javadoc. 

```suggestion
     * @param nodeId the node instances id that should be triggered
```

Reply:
done